__tests__/index.test.js

```diff
@@ -69,15 +69,15 @@ describe('cli', () => {
     });
 
     it('should print usage for a given command', async () => {
-      await expect(cli(['swagger', '--help'])).resolves.toMatchSnapshot();
+      await expect(cli(['openapi', '--help'])).resolves.toMatchSnapshot();
     });
 
     it('should print usage for a given command if supplied as `help <command>`', async () => {
-      await expect(cli(['help', 'swagger'])).resolves.toMatchSnapshot();
+      await expect(cli(['help', 'openapi'])).resolves.toMatchSnapshot();
     });
 
     it('should not surface args that are designated as hidden', async () => {
-      await expect(cli(['swagger', '--help'])).resolves.toMatchSnapshot();
+      await expect(cli(['openapi', '--help'])).resolves.toMatchSnapshot();
```

Comment:
I figured it makes more sense to test our `openapi` command rather than our deprecated `swagger` command here, which resulted in a bunch of snapshot updates in `__tests__/__snapshots__/index.test.js.snap`.